app/frontend/modules/filter_toggle.js

```diff
@@ -0,0 +1,112 @@
+export class FilterToggle {
+  constructor (options) {
+    this.options = options
+    this.container = this.options.toggleButton.container
+  }
+
+  setupResponsiveChecks () {
+    this.mq = window.matchMedia(this.options.bigModeMediaQuery)
+    this.mq.addListener(this.checkMode.bind(this))
+    this.checkMode(this.mq)
+  }
+
+  checkMode (mq) {
+    if (mq.matches) {
+      this.enableBigMode()
+    } else {
+      this.enableSmallMode()
+    }
+  }
+
+  enableBigMode () {
+    this.showMenu()
+    this.removeMenuButton()
+    this.removeCloseButton()
+  }
+
+  enableSmallMode () {
+    this.options.filter.container.setAttribute("tabindex", "-1")
+    this.hideMenu()
```

Comment:
If we open the side menu in small mode, resize it to large and immediately resize it back to small, it'll hide the side menu. Is that intended?

Reply:
Yeah, I think that’s fine (one of those things where not a right answer). It’s kind of an edge case (though one that may happen if you’re on a table and rotate it), but I think the most important thing is that the filters don’t disappear on desktop widths, which is what’s happening at the moment.